Packs/FlashpointFeed/Integrations/FlashpointFeed/FlashpointFeed.py

```diff
@@ -0,0 +1,407 @@
+}
+
+HTTP_ERRORS = {
+    400: "Bad request: An error occurred while fetching the data.",
+    401: "Authentication error: Please provide valid API Key.",
+    403: "Forbidden: Please provide valid API Key.",
+    404: "Resource not found: Invalid endpoint was called.",
+    500: "Internal server error: Please try again after some time."
+}
+
+INTEGRATION_VERSION = "v1.0.0"
+INTEGRATION_PLATFORM = "XSOAR Cortex"
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API"""
+
+    def http_request(self, url_suffix, method="GET", params=None) -> Any:
+        """
+        Get http response based on url and given parameters.
+
+        :param method: Specify http methods
+        :param url_suffix: url encoded url suffix
+        :param params: None
+
+        :return: http response on json
+        """
+        resp = self._http_request(
+            method=method,
+            url_suffix=url_suffix,
+            params=params,
+            ok_codes=(200, 201),
+            error_handler=self.handle_errors,
+            timeout=TIMEOUT
+        )
+
+        return resp
+
+    def check_indicator_type(self, indicator_value: str, default_map: bool) -> str:
+        """
+        Function to set the type of the indicator.
+
+        :param indicator_value: Value of the indicator.
+        :param default_map: To enable the default mapper setting for the indicator.
+
+        :return: Type of the indicator.
+        """
+        ind_type = DEFAULT_INDICATOR_TYPE
+        if not default_map:
+            ind_type = auto_detect_indicator_type(indicator_value)
+
+            if not ind_type:
+                ind_type = DEFAULT_INDICATOR_TYPE
+
+        return ind_type
+
+    def create_relationship(self, entity_a: str, entity_a_type: str, tags: list) -> List:
+        """
+        Create a list of relationships objects from the tags.
+
+        :param entity_a: the entity a of the relation which is the current indicator.
+        :param entity_a_type: the entity a type which is the type of the current indicator.
+        :param tags: list of tags returned from the API.
+
+        :return: list of EntityRelationship objects containing all the relationships.
+        """
+        if not tags:
+            return []
+
+        my_tags = [tag for tag in tags if tag.get('is_galaxy')]
+        relationships = []
+        for tag in my_tags:
+            entity_b = tag.get('name')
+            if entity_b:
+                if 'misp-galaxy:mitre-enterprise-attack' in entity_b:
+                    name = entity_b.split("=")
+                    entity_b = name[1].replace('\"', "")
+
+                obj = EntityRelationship(
+                    name=EntityRelationship.Relationships.INDICATOR_OF,
+                    entity_a=entity_a,
+                    entity_a_type=entity_a_type,
+                    entity_b=entity_b,
+                    entity_b_type='Attack Pattern'
```

Comment:
The Indicator type in entity_b_type will not always be of the `Attack Pattern` type, there may be other Threat Intel Objs such as Malware, Tool, Intrusion Set etc.

I think it's supposed to look like this:
"misp-galaxy:mitre-enterprise-attack-attack-pattern=........"
"misp-galaxy:mitre-enterprise-attack-Malware=......"
"misp-galaxy:mitre-enterprise-attack-Tool=........"

Please check it out.